src/ext/java/org/opentripplanner/ext/vectortiles/layers/stops/DigitransitRealtimeStopPropertyMapper.java

```diff
@@ -32,10 +34,19 @@ protected Collection<KeyValue> map(RegularStop stop) {
       .stream()
       .anyMatch(alert -> alert.noServiceAt(currentTime));
 
+    var serviceDate = LocalDate.ofInstant(currentTime, transitService.getTimeZone());
+    boolean stopTimesExist = transitService
+      .getStopTimesForStop(stop, serviceDate, ArrivalDeparture.BOTH, true)
+      .stream()
+      .anyMatch(stopTime -> stopTime.times.size() > 0);
+
     Collection<KeyValue> sharedKeyValues = getBaseKeyValues(stop, i18NStringMapper, transitService);
     return ListUtils.combine(
       sharedKeyValues,
-      List.of(new KeyValue("closedByServiceAlert", noServiceAlert))
+      List.of(
+        new KeyValue("closedByServiceAlert", noServiceAlert),
+        new KeyValue("noServiceOnServiceDay", !stopTimesExist)
```

Comment:
Would it be possible to turn this around? Basically you state what exists (services) and not what doesn't exist?

Reply:
Double negatives (noService=false) always make my head hurt.

Reply:
Of course! Sorry about that.